File: Arrays/Practice/minimum.c
#include <stdio.h>

int main() {

    int arr[] = {10, 25, 7, 40, 15};
    int n = 5;

    int minimum = arr[0];

    for (int i = 1; i < n; i++) {

        if (arr[i] < minimum) {
            minimum = arr[i];
        }
    }

    printf("Minimum element = %d\n", minimum);

    return 0;
}

File: Arrays/Practice/minimum.java
package Practice;
public class minimum {

    public static void main(String[] args) {

        int[] arr = {10, 25, 7, 40, 15};

        int minimum = arr[0];

        for (int i = 1; i < arr.length; i++) {

            if (arr[i] < minimum) {
                minimum = arr[i];
            }
        }

        System.out.println("Minimum element = " + minimum);
    }
}